plugins/ecs-bridge/commands/commands.go

```diff
@@ -135,19 +150,18 @@ func del(args *skel.CmdArgs, engine engine.Engine) error {
 	if err != nil {
 		return err
 	}
-	detailLog("Deleting veth interface", args, conf, "")
+	detailLogInfo("Deleting veth interface", args, conf, "")
 
 	if utils.ZeroOrNil(conf.IPAM) {
-		detailLog("IPAM configuration not found, skip DEL for IPAM", args, conf, "")
-		return nil
-	}
-
-	detailLog("Running IPAM plugin DEL", args, conf, "")
-	err = engine.RunIPAMPluginDel(conf.IPAM.Type, args.StdinData)
-	if err != nil {
-		return err
+		detailLogInfo("IPAM configuration not found, skip DEL for IPAM", args, conf, "")
```

Comment:
Agreed, but I think in that case there might be two issues within the `del` function. This PR is fixing one of the two (skipping VethDelete). As for the the other issue, it seems like we might need to synchronize on what makes sense for the agent to pass to the cni plugins.

That being said, it's still not clear to me how the ipam cni plugin fits into this, since it never logs anything.